gatling/user-files/simulations/HttpGetDocument.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2011-2018 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package computerdatabase
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.concurrent.duration._
+import sys.process._
+
+class HttpGetDocument extends Simulation {
+  val host = System.getProperty("host", "localhost")
+  val requests = System.getProperty("requests", "2000").toInt
+  val users = System.getProperty("users", "1").toInt
+  val duration = System.getProperty("duration", "1").toInt
+  var jwt = System.getProperty("jwt", "some jwt")
+
+  val result = Process("""node ./user-files/utils/requestOneId""")
```

Comment:
nitpicking: I don't think we need a raw string interpolator here

```suggestion
  val result = Process("node ./user-files/utils/requestOneId")
```